python/tvm/relay/frontend/tflite.py

```diff
@@ -224,6 +224,20 @@ def has_same_qnn_params(self, lhs_tensor, rhs_tensor):
         return lhs_tensor.qnn_params['scale'] == rhs_tensor.qnn_params['scale'] and \
                 lhs_tensor.qnn_params['zero_point'] == rhs_tensor.qnn_params['zero_point']
 
+    def is_qnn_op(self, op):
```

Comment:
Maybe we should be better name it as `is_quantized`, because `qnn` is our relay concept. But here we accept tflite op.

Reply:
Done